sdk/python/feast/permissions/auth/kubernetes_token_parser.py

```diff
@@ -41,10 +42,14 @@ async def user_details_from_access_token(self, access_token: str) -> User:
         current_user = f"{sa_namespace}:{sa_name}"
         logging.info(f"Received request from {sa_name} in {sa_namespace}")
 
-        roles = self.get_roles(sa_namespace, sa_name)
-        logging.info(f"SA roles are: {roles}")
+        skip_auth_key = os.getenv("SKIP_AUTH_KEY")
+        if sa_name is not None and sa_name == skip_auth_key:
+            return User(username=current_user, roles=[])
```

Comment:
so we're using a User with the same name as the SKIP_AUTH_KEY env var and no roles?
1- The `if` condition is `True` if all the servers use the same value of this env var
2- If we add a User with no roles, the permission enforcement probably fails

Reply:
2 - what do you mean ? there is no permission enforcement for "skip_auth_key" user ? or am I missing something ?

Reply:
Oh now I see, it was a bit hidden under that long if. So, if `1-` is satisfied, there is no enforcement 
(BTW: why not adding a `is_intra_service_user()` method to `User` instead?)

Reply:
I don't have a user object should I add a static method to User or add a utils file, I think the second option is better or leave it be what do you say ?